git_cookbook.md

```diff
@@ -20,3 +20,23 @@ function pr () {
   open https://github.com/$repo/compare/$branch?expand=1
 }
 ```
+
+#### A less verbose status command
+Includes only the files that changed and omits the boilerplate about the branch, etc.  Very handy to alias as `git s`.
+```
+git status -s
+``` 
+
+#### A better overview of history
+This command lists commits one-per-line, in color, with a graph of the history on the left.  Very handy for quickly looking at history or checking the relationships between branches.  I alias it as `git lg`.
+
+```
+git log --oneline --decorate --graph --all -100
```

Comment:
I use `less`.  The issue isn't the viewing, it's the generating.  Even if you don't look at all of it, it has to assemble it all, and `less` won't show you the contents until the stream closes (AFAICT).  I'll add a note.